todo_app/tests/test_app.py

```diff
@@ -1,71 +0,0 @@
-import pytest, requests, os, json
```

Comment:
I know we talked about these, so happy enough they're deleted. Just in case you're interested, fixing these might look like below, combined with shifting the import in `mongodb_items` to `import pymongo` from `from pymongo import MongoClient`:
```python
import pytest
from dotenv import load_dotenv, find_dotenv
from todo_app import app
import mongomock

from todo_app.data.mongodb_items import add_item

@pytest.fixture
def client():
    file_path = find_dotenv('.env.test')
    load_dotenv(file_path, override=True)

    with mongomock.patch(servers=(('fakemongo.com', 27017),)):
        test_app = app.create_app()

        with test_app.test_client() as client:
            yield client

def test_index_page_gets_items(client):
    add_item('Test card')
    response = client.get('/')

    assert response.status_code == 200
    assert 'Test card' in response.data.decode()

def test_add_page_route(client):
    response = client.post('/add')

    assert response.status_code == 302

def test_complete_page_route(client):
    response = client.post('/complete')

    assert response.status_code == 302

def test_edit_page_route(client):
    response = client.post('/edit')

    assert response.status_code == 302

def test_delete_page_route(client):
    response = client.post('/delete')

    assert response.status_code == 302
```